batchflow/models/torch/blocks/named_blocks.py

```diff
@@ -120,23 +112,77 @@ def __init__(self, inputs=None, layout='cnacn', channels='same', kernel_size=3,
             # If main flow changes the number of channels, so must do the side branch.
             # No activation, because it will be applied after summation with the main flow
             branch_params = {'layout': branch_layout, 'channels': channels[-1],
-                             'kernel_size': 1, 'stride': branch_stride_downsample}
+                             'kernel_size': 1, 'stride': branch_stride_downsample, 'groups': branch_groups}
         else:
-            branch_params = {}
+            branch_params = {'stride': branch_stride_downsample, 'groups': branch_groups}
```

Comment:
What's about providing `branch_layout` here too?

Reply:
It would change default branch behaviour in case of no downsampling from `Identity()` to `cn`, but I can fix it